__tests__/unit/mock-backend-response.test.js

```diff
@@ -26,76 +35,62 @@ describe('Render Backend Response', () => {
 
 	test('calls next if request is not GET or POST', () => {
 		const middleware = mockBackendResponse(options);
-		const next = jest.fn();
 
 		middleware({
 			method: 'head',
-			url: '/a/url'
+			path: '/a/url'
 		}, undefined, next);
 
 		expect(next).toHaveBeenCalledWith();
+		expect(next.mock.calls[0].length).toEqual(0);
 	});
 
-	test('opens a mock file', () => {
-		const {readFile} = require('fs');
+	test('calls next if request does not match any mock files', () => {
 		const middleware = mockBackendResponse(options);
-		const request = {
-			method: 'get',
-			url: '/a/url'
-		};
-
-		middleware(request);
 
-		expect(readFile).toHaveBeenCalledWith(`${options.directory}${request.url}_get.json`, expect.any(Function));
-	});
-
-	test('calls next if there was an error opening the file', () => {
-		const {__getReadFileCallback} = require('fs');
-		const middleware = mockBackendResponse(options);
-		const request = {
+		middleware({
 			method: 'get',
-			url: '/a/url'
-		};
-		const next = jest.fn();
-		middleware(request, undefined, next);
-
-		__getReadFileCallback()({error: 'not found'});
+			path: '/does/not/exist'
+		}, undefined, next);
 
 		expect(next).toHaveBeenCalledWith();
+		expect(next.mock.calls[0].length).toEqual(0);
 	});
```

Comment:
Isn't it two way of testing the same thing, i.e that `next` have been called with no arguments?

Reply:
Yes, this is redundant!